vault/resource_pki_secret_backend_role_test.go

```diff
@@ -12,6 +12,17 @@ import (
 	"github.com/hashicorp/terraform-provider-vault/testutil"
 )
 
+var legacyPolicyIdentifiers = `policy_identifiers = ["1.2.3.4"]`
```

Comment:
Since the `vault` project is already so massive, I'd suggest that we either move theses vars within the test's scope, or include `test` in their names.

Reply:
:+1: